packages/dso-toolkit/libs/bootstrap/_thumbnails.scss

```diff
@@ -16,7 +16,7 @@
 
   > img,
   a > img {
-    @include img-responsive;
+    // @include img-responsive;
```

Comment:
Ik heb deze nu uitgezet omdat de mixin is verplaatst naar /sources/, maar deze hele styling wordt nu niet door de toolkit gebruikt. Wat vind jij @tfrijsewijk? Deze gehele file maar meteen opschonen (niet verhuizen)?

Reply:
Na overleg is besloten om dit soort files te deleten